arrows/vtk/applets/color_mesh.cxx

```diff
@@ -238,6 +235,15 @@ class color_mesh::priv : public kwiver::arrows::vtk::mesh_coloration
       return FAIL;
     }
 
+    // set variables from the config
+    frame_sampling_ = config_->get_value("frame_sampling", 1);
+    frame_ = config_->get_value("frame", -1);
+    all_frames_ = config_->get_value("all_frames", false);
+    occlusion_threshold_ = config_->get_value("occlusion_threshold", 0.0);
+    color_occluded_ = config_->get_value("color_occluded", false);
+    color_masked_ = config_->get_value("color_masked", false);
+    remove_color_count_less_equal_ = config_->get_value("remove_color_count_less_equal", true);
```

Comment:
The default value needs to be update since this is no longer boolean.

More generally, the way we usually get config values is to use the variable itself as a default.  This way you don't need to keep these default values in sync in two places in the code.  So do this instead
```suggestion
    remove_color_count_less_equal_ = config_->get_value("remove_color_count_less_equal", 
                                                        remove_color_count_less_equal_);
```

and likewise for the other config settings